src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.props

```diff
@@ -49,6 +49,7 @@
   <Import Project="Microsoft.Android.Sdk.DefaultItems.props" />
 
   <PropertyGroup>
+    <XamarinAndroidVersion>$(AndroidNETSdkVersion)</XamarinAndroidVersion>
```

Comment:
It may make sense to move this into https://github.com/xamarin/xamarin-android/blob/47c260816e4a74304f0cad25d348eb2926a43de1/build-tools/create-packs/Microsoft.Android.Sdk.proj#L104 just to keep these version props together, though I'm not sure if that would have any other downside.